src/app/components/SocialLogin.jsx

```diff
@@ -1,9 +1,14 @@
 import Image from "next/image";
 import styles from "../stylesheet/components/SocialLogin.module.css";
+import { usePathname } from 'next/navigation'
 
 export default function SocialLoging() {
+  const router = usePathname();
+  const isSignUpPage = router === '/sign-up';
+  const buttonText = isSignUpPage ? 'Sign Up' : 'Sign In';
+
   return (
-    <div className={styles.container}>
+    <div className={isSignUpPage ? styles.container : styles['container--sign-in']}>
```

Comment:
Same as previous comment, you need to include the original class when using a modifier class
```suggestion
    <div className={`${styles.container}${isSignUpPage ? '' : ' ' + styles['container--sign-in']}`}>
```